CreateSnapshot/src/main/java/org/opensearch/migrations/CreateSnapshot.java

```diff
@@ -74,7 +74,10 @@ public static class S3RepoInfo {
 
     public static void main(String[] args) throws Exception {
         Args arguments = new Args();
-        JCommander jCommander = JCommander.newBuilder().addObject(arguments).build();
+        JCommander jCommander = JCommander.newBuilder()
+            .allowParameterOverwriting(true)
```

Comment:
I don't like this idea at all - we should throw.  As a user of these programs, I sometimes have long invocations and I might hop to the beginning or end of the list because it's convenient.  Allowing the user to put two in but then just use one means that you're ignoring something that the user tried to pass in.

Reply:
Removed